experimental/BasisLieHighestWeight/src/MonomialBasis.jl

```diff
@@ -35,6 +35,14 @@ monomial_ordering(basis::MonomialBasis) = basis.monomial_ordering
 
 birational_sequence(basis::MonomialBasis) = basis.birational_seq
 
+function volume_of_polytope(basis::MonomialBasis)
+  if has_attribute(basis, :volume_of_polytope)
+    return get_attribute(basis, :volume_of_polytope)
```

Comment:
```suggestion
    return get_attribute(basis, :volume_of_polytope)::SomeType
```#
this needs a manual type annotation, as the attribute storage does not keep track of types